hammerblade/torch/tests/hbutils.py

```diff
@@ -0,0 +1,6 @@
+def init_hb_tensor(input_t):
+    """
+    Returns a HB tensor that's a leaf node of the computation graph.
+    """
+    return input_t.hammerblade().clone().detach().requires_grad_(
```

Comment:
This is very smart! apparently I don't know much about using pytorch and ML in general. ;)